model/project_ref.go

```diff
@@ -1037,6 +1036,7 @@ func (p *ProjectRef) createNewRepoRef(u *user.DBUser) (repoRef *RepoRef, err err
 	repoRef.Id = mgobson.NewObjectId().Hex()
 	repoRef.RepoRefId = ""
 	repoRef.Identifier = ""
+	repoRef.Enabled = false
```

Comment:
False is the default value; we shouldn't need this.

Reply:
done